week-3/Homework/E-array-map/exercise.js

```diff
@@ -3,3 +3,9 @@
 
 var numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
+function multiple100(number) {
+  return number * 100;
+}
+var numbersMultiplied = numbers.map(multiple100);
+
+console.log(numbersMultiplied);
```

Comment:
You should write code to cover the other examples. It's important to understand the different ways functions are expressed and used in Javascript.

Reply:
Thanks for your feedback. I forgot to write different ways. I will add it asap. Thank you so much.